<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreprocessWindow</class>
 <widget class="QMainWindow" name="PreprocessWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1618</width>
    <height>1109</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>电压范围</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_2">
          <item>
           <property name="text">
            <string>-50uV~50uV</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>-100uV~100uV</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>-200uV~200uV</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>时间轴范围</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox">
          <item>
           <property name="text">
            <string>0~1s</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>0~5s</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>0~10s</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>&lt;&lt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>&gt;&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>起始时间：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>秒</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_20">
          <property name="text">
           <string>结束时间：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_2"/>
        </item>
        <item>
         <widget class="QLabel" name="label_21">
          <property name="text">
           <string>秒</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_4">
          <property name="text">
           <string>跳转</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_5">
          <property name="text">
           <string>波形重叠</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>返回上一级菜单</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget">
        <property name="minimumSize">
         <size>
          <width>1400</width>
          <height>900</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>基本信息</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="maximumSize">
             <size>
              <width>135</width>
              <height>44</height>
             </size>
            </property>
            <property name="text">
             <string>电极数量：</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="maximumSize">
             <size>
              <width>134</width>
              <height>44</height>
             </size>
            </property>
            <property name="text">
             <string>-</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="maximumSize">
             <size>
              <width>135</width>
              <height>44</height>
             </size>
            </property>
            <property name="text">
             <string>事件数量：</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="maximumSize">
             <size>
              <width>134</width>
              <height>44</height>
             </size>
            </property>
            <property name="text">
             <string>-</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="maximumSize">
             <size>
              <width>135</width>
              <height>44</height>
             </size>
            </property>
            <property name="text">
             <string>Epoch数量：</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_9">
            <property name="maximumSize">
             <size>
              <width>134</width>
              <height>44</height>
             </size>
            </property>
            <property name="text">
             <string>-</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="maximumSize">
             <size>
              <width>135</width>
              <height>44</height>
             </size>
            </property>
            <property name="text">
             <string>采样率：</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_11">
            <property name="maximumSize">
             <size>
              <width>134</width>
              <height>44</height>
             </size>
            </property>
            <property name="text">
             <string>-</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_12">
            <property name="maximumSize">
             <size>
              <width>135</width>
              <height>44</height>
             </size>
            </property>
            <property name="text">
             <string>总时长：</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_13">
            <property name="maximumSize">
             <size>
              <width>134</width>
              <height>44</height>
             </size>
            </property>
            <property name="text">
             <string>-</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_14">
            <property name="maximumSize">
             <size>
              <width>135</width>
              <height>44</height>
             </size>
            </property>
            <property name="text">
             <string>重参考：</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_15">
            <property name="maximumSize">
             <size>
              <width>134</width>
              <height>44</height>
             </size>
            </property>
            <property name="text">
             <string>-</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_16">
            <property name="maximumSize">
             <size>
              <width>135</width>
              <height>44</height>
             </size>
            </property>
            <property name="text">
             <string>电极定位：</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_17">
            <property name="maximumSize">
             <size>
              <width>134</width>
              <height>44</height>
             </size>
            </property>
            <property name="text">
             <string>-</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1618</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>加载数据</string>
    </property>
    <widget class="QMenu" name="menu_2">
     <property name="title">
      <string>从外界...</string>
     </property>
     <addaction name="actionedf"/>
     <addaction name="actioneeg_2"/>
    </widget>
    <addaction name="actionlocalFile"/>
    <addaction name="menu_2"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>数据分析</string>
    </property>
    <addaction name="actionPSD"/>
    <addaction name="actionDWT"/>
    <addaction name="actionWigner"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>预处理</string>
    </property>
    <addaction name="actionDJDW"/>
    <addaction name="actionFIR"/>
    <addaction name="actionspilt"/>
    <addaction name="actionre_sampling"/>
    <addaction name="actionbaseline"/>
   </widget>
   <widget class="QMenu" name="menu_7">
    <property name="title">
     <string>工具</string>
    </property>
    <addaction name="actionrenameMotage"/>
    <addaction name="actionbadData"/>
    <addaction name="actionbadChannel"/>
    <addaction name="actionICA"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
   <addaction name="menu_7"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionlocalFile">
   <property name="text">
    <string>从本程序...</string>
   </property>
  </action>
  <action name="actionedf">
   <property name="text">
    <string>.edf/.edf+</string>
   </property>
  </action>
  <action name="actionbdf">
   <property name="text">
    <string>bdf</string>
   </property>
  </action>
  <action name="actioneeg">
   <property name="text">
    <string>eeg</string>
   </property>
  </action>
  <action name="actioneeg_2">
   <property name="text">
    <string>.eeg(brainvision)</string>
   </property>
  </action>
  <action name="actionFIR">
   <property name="text">
    <string>滤波（FIR）</string>
   </property>
  </action>
  <action name="actionhighPass">
   <property name="text">
    <string>高通</string>
   </property>
  </action>
  <action name="actionnotch">
   <property name="text">
    <string>陷波</string>
   </property>
  </action>
  <action name="actionbadChannel">
   <property name="text">
    <string>插值坏导</string>
   </property>
  </action>
  <action name="actionbadData">
   <property name="text">
    <string>剔除坏段</string>
   </property>
  </action>
  <action name="actionresampling">
   <property name="text">
    <string>降采样</string>
   </property>
  </action>
  <action name="actionautoRejection">
   <property name="text">
    <string>自动剔除</string>
   </property>
  </action>
  <action name="actionICA">
   <property name="text">
    <string>ICA剔除伪迹</string>
   </property>
  </action>
  <action name="actionrenameMotage">
   <property name="text">
    <string>电极重命名</string>
   </property>
  </action>
  <action name="actioneeg_vhdr">
   <property name="text">
    <string>eeg(.vhdr)</string>
   </property>
  </action>
  <action name="actionPSD">
   <property name="text">
    <string>PSD（功率谱估计）</string>
   </property>
  </action>
  <action name="actionDJDW">
   <property name="text">
    <string>电极定位</string>
   </property>
  </action>
  <action name="actionspilt">
   <property name="text">
    <string>数据分段</string>
   </property>
  </action>
  <action name="actionre_sampling">
   <property name="text">
    <string>重参考</string>
   </property>
  </action>
  <action name="actionbaseline">
   <property name="text">
    <string>基线校正</string>
   </property>
  </action>
  <action name="actionDWT">
   <property name="text">
    <string>DWT（小波变换）</string>
   </property>
  </action>
  <action name="actionWigner">
   <property name="text">
    <string>WVD（维格纳-费力分布）</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
